### DISCUSSION TITLE
Exporting multiple plates at once

### DISCUSSION THREAD

**Discussion by kabriolin60** [OrcaSlicer/OrcaSlicer#2676] [Q&A]
Hi all Orca users,

I was wondering if there is a solution/feature to exporte multiple plates at once. 
We can slice 1 to N plates with one click, but (as far as I know) there is option for exporting (and uploading) multiple plates at once.

Do you guys know if the feature already exists or not?
Thank you,
Kabriolin

**Comment by czyz** [comment]
There is a 'print plate' and a 'print all' function, and I'd love to see the 'print all' function (if vestigial) renamed to 'print all plates' and have the ability to print multiple plates using systems like the Swapmod and Chitu Systems PlateCycler C1M. 

I speculate but I think the steps to do this would be:
1. Add a "Plate change G-code" field to the "Machine G-code" tab of the printer's "Printer Settings".
2. Have Orcaslicer generate a 3mf file that injects that plate change code in between each plate.

Both Swapmod and the Chitu C1M use additional software to inject and create these 3mf files that include multiple plates, but I'd love to see this just be a built-in feature of Orcaslicer. It'd be usable with all manner of plate cycling and swapping devices due to the custom g-code field. 
 
This seems like a related discussion: https://github.com/OrcaSlicer/OrcaSlicer/discussions/446

**Comment by czyz** [comment]
I added this feature to a fork of OrcaSlicer and submitted this PR.
https://github.com/OrcaSlicer/OrcaSlicer/pull/13177
